README.md

```diff
@@ -5,4 +5,22 @@ Web app for checking tmt tests, plans and stories
 2. Install the requirements
 3. Use the `start_api.sh` script to start the api
 ## Tests
-To run the tests, use the pytest command
\ No newline at end of file
+To run the tests, use the pytest command
+
+# Requirements
+## As a user I want to have a clickable link to test/plan verifying the issue
+Covered by this [issue](https://issues.redhat.com/browse/TT-207). Implemented by creating an API, that will upon request clone the git repository, 
+parse the test/plan and return metadata in mainly machine-readable format (but a basic `html` output should
```

Comment:
The machine readable part sounds rather like an implementation detail. From the user perspective it is important the data are presented in a human readable format.